scampijoes_stuff1.md

```diff
@@ -0,0 +1,6 @@
+# Some ideas
+
+## Dinner
+
+* Zur Post: simple but delicious restaurant in Reifenberg - do not forget a reservation
```

Comment:
Naiv in Frankfurt: Good collection of CraftBeer 🍺 and delicious bistro kitchen + good service